client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -199,7 +199,9 @@ void main(void)
 #endif
 
 	if (fogDistance != 0.0) {
-		float d = clamp((fogDistance - length(eyeVec)) / (fogDistance * 0.6), 0.0, 1.0);
+		// On some systems, the order of operations matters here.
+		// See the commit message of the commit that changed this.
+		float d = clamp(1.0 / 0.6 - length(eyeVec) / (fogDistance * 0.6), 0.0, 1.0);
```

Comment:
@paramat: Changed to 1.7, which is probably just as good as 1.66667.

The code now reads: `1.7 - 1.7 * length(eyeVec) / figDistance`

Is that OK with you ?

IMO the exact value is not paramount. `1.0 / 0.6` makes the first fog start at 40% of the fog distance, 1.7 makes it start at 41%. Nobody would notice.